script/workScript.js

```diff
@@ -0,0 +1,151 @@
+const workData = [
+  {
+    id: 1,
+    name: 'Multi-Post Stories Gain+Glory',
+    photo: './Images/SecondPortfolio.svg',
+    languages_list: ['Ruby on rails', 'CSS', 'Javascript', 'Html'],
+    btn_text: 'See Project',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry Lorem Ipsum has been the industry s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been theindustry's standard dummy text ever since the 1500s,when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    heading: 'Keeping track of hundreds of components',
+  },
+  {
+    id: 2,
+    name: 'Multi-Post Stories Gain+Glory',
+    photo: './Images/SecondPortfolio.svg',
+    languages_list: ['Ruby on rails', 'CSS', 'Javascript', 'Html'],
+    btn_text: 'See Project',
+  },
+  {
+    id: 3,
+    name: 'Multi-Post Stories Gain+Glory',
+    photo: './Images/SecondPortfolio.svg',
+    languages_list: ['Ruby on rails', 'CSS', 'Javascript', 'Html'],
+    btn_text: 'See Project',
+  },
+  {
+    id: 4,
+    name: 'Multi-Post Stories Gain+Glory',
+    photo: './Images/SecondPortfolio.svg',
+    languages_list: ['Ruby on rails', 'CSS', 'Javascript', 'Html'],
+    btn_text: 'See Project',
+  },
+  {
+    id: 5,
+    name: 'Multi-Post Stories Gain+Glory',
+    photo: './Images/SecondPortfolio.svg',
+    languages_list: ['Ruby on rails', 'CSS', 'Javascript', 'Html'],
+    btn_text: 'See Project',
+  },
+  {
+    id: 6,
+    name: 'Multi-Post Stories Gain+Glory',
+    photo: './Images/SecondPortfolio.svg',
```

Comment:
- [ ] It looks like all the data are exactly the same on the script, to use different information for each project make it easier to see bugs and have a project more like it should be work, I invite you to change the info and fix the necessary issues on your project after that.